### PR TITLE
Add information about the VO property

### PR DESCRIPTION
**Pull request by enolfc**
# Summary

Document the use of the `VO` property so discovery can properly work with cloud-info-provider

---

**Related issue :**

### CODE REVIEW

**Review by gwarf — APPROVED** [review]
LGTM

**Comment by gwarf** [comment]
@CatalinCondurache , please have a look

**Comment by sebastian-luna-valero** [comment]
Shall we promote `egi_VO` instead as discussed in the FedCloud slack channel?

**Comment by sebastian-luna-valero** [comment]
On the meeting on 24th Oct 2023 it was decided (and tested) to use `egi.VO`

**Comment by enolfc** [comment]
> On the meeting on 24th Oct 2023 it was decided (and tested) to use `egi.VO`

yes needs update

File: content/en/providers/cloud-compute/openstack/vo_config.md [hunk 1]
```diff
@@ -15,18 +15,24 @@ The usual method of supporting a VO is by creating a local project for it. You
...
    ```
 
-1. Add that group to the desired local project: :
+1. Add that group to the desired local project:
 
    ```shell
    $ openstack role add member --group $group_id --project <your project>
    ```
 
+1. Set the `VO` property to the name of the VO that you are supporting:
```
Comment (sebastian-luna-valero) [inline content/en/providers/cloud-compute/openstack/vo_config.md:30]:
```suggestion
1. Set the `egi.VO` property to the name of the VO that you are supporting:
```

File: content/en/providers/cloud-compute/openstack/vo_config.md [hunk 2]
```diff
@@ -15,18 +15,24 @@ The usual method of supporting a VO is by creating a local project for it. You
...
+
+   ```shell
+   $ openstack project set --property VO=<name of the VO> <your project>
```
Comment (sebastian-luna-valero) [inline content/en/providers/cloud-compute/openstack/vo_config.md:33]:
```suggestion
   $ openstack project set --property egi.VO=<name of the VO> <your project>
```

File: content/en/providers/cloud-compute/openstack/vo_config.md [hunk 3]
```diff
@@ -105,16 +111,8 @@ openstack role add member --user <your caso user> --project <your new vo project
...
-[fedcloud-catchall-operations repository](https://github.com/EGI-Federation/fedcloud-catchall-operations)
-
-```yaml
----
-vos:
-  - name: <vo name>
-    auth:
-      project_id: <your new vo project>
-```
+If you are correctly setting the `VO` property to your projects, the configuration will
```
Comment (sebastian-luna-valero) [inline content/en/providers/cloud-compute/openstack/vo_config.md:114]:
```suggestion
If you are correctly setting the `egi.VO` property to your projects, the configuration will
```

**Review by sebastian-luna-valero — CHANGES_REQUESTED** [review]
Let's update the property name from `VO` to `egi.VO` as agreed with fedcloud providers in the fedcloud task force meeting on 24th Oct 2023.

**Comment by enolfc** [comment]
@CatalinCondurache can you check this one?

**Review by CatalinCondurache — APPROVED** [review]
approved

**Comment by gwarf** [comment]
Thanks, @CatalinCondurache.
@sebastian-luna-valero the merge is blocked by your request for changes, can you please check/confirm all is OK?

**Review by sebastian-luna-valero — APPROVED** [review]

**Pull request merged**